client/src/components/LibraryNavigator/const.ts

```diff
@@ -4,6 +4,9 @@ import { l10n } from "vscode";
 
 export const Messages = {
   TableDeletionError: l10n.t("Unable to delete table {tableName}."),
+  TablesDeletionWarning: l10n.t(
+    "Are you sure you want to delete {count} table(s): {tableNames}?",
```

Comment:
It might be worth asking @skyeaussie what message to use here. We may not need the count or extra parens around the s. Maybe something like...
```
Are you sure you want to delete the following tables: {tableNames}?
```
It looks like studio uses the following two messages: 
```
Are you sure you want to delete the selected tables?
Are you sure you want to delete the table "{tableName}"
```